### PR TITLE
Add db image, format code, add management script

### PR DESCRIPTION
**Pull request by WesleyKlop**
```text
ipsenh on  feature/cleanup took 3s 
❯ ./journali.sh help
      _                              _ _ 
     | |                            | (_)
     | | ___  _   _ _ __ _ __   __ _| |_ 
 _   | |/ _ \| | | | '__| '_ \ / _` | | |
| |__| | (_) | |_| | |  | | | | (_| | | |
 \____/ \___/ \__,_|_|  |_| |_|\__,_|_|_|

Available commands:
 - init: Initialize the child repositories
 - install: Intall frontend node dependencies
 - migrate: Migrate the database
 - start: Start the development environment
 - stop: Stop the development environment
 - yarn [command]: Run any yarn command on the frontend environment
 - cargo [command]: Run any cargo command on the api environment
 - help: This command

```

Also does other cleanup of things like normalizing names, not exposing ports we don't have to, and upgrading the builder compose to version 3

### CODE REVIEW

File: journali.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    start)
+        echo " - Starting development environment"
+        docker-compose up -d
+        ;;
+    stop)
+        echo " - Stopping development environment"
+        docker-compose down
+        ;;
+    yarn | cargo)
+        echo "Running: "$BASE_COMMAND ${@:1}
```
Comment (DutchGhost) [inline journali.sh:36]: `echo " - running: ..."` to match the other echo's in all other cases

File: journali.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ - install: Intall frontend node dependencies
+ - migrate: Migrate the database
+ - start: Start the development environment
+ - stop: Stop the development environment
+ - yarn [command]: Run any yarn command on the frontend environment
+ - cargo [command]: Run any cargo command on the api environment
+ - help: This command"
+        ;;
+    *)
+        echo "Unknown command \"$1\""
```
Comment (DutchGhost) [inline journali.sh:51]: `echo " - Unknown command"` to match all other echo's

Comment (WesleyKlop) [reply journali.sh:51]: Nah because it's an error message I don't think we should do that

**Comment by WesleyKlop** [comment]
> Mist alleen nog newlines aan het einde van dev.env en journali.sh.

Fixed

File: journali.sh (hunk 2 above)
Comment (WesleyKlop) [reply journali.sh]: I changed it to send things to stderr does that help

**Pull request merged**